core/src/main/java/io/kestra/core/serializers/JacksonMapper.java

```diff
@@ -124,4 +129,32 @@ private static ObjectMapper configure(ObjectMapper mapper) {
             .registerModules(new GuavaModule())
             .setTimeZone(TimeZone.getDefault());
     }
+
+    private static ObjectMapper createIonObjectMapper() {
+        return configure(new IonObjectMapper(new IonFactory(createIonSystem())))
+            .setSerializationInclusion(JsonInclude.Include.ALWAYS)
+            .registerModule(new IonModule());
+    }
+
+    /**
+     * @see IonSystemBuilder#build()
+     */
+    // TODO simplify after https://github.com/amazon-ion/ion-java/pull/781
+    private static IonSystem createIonSystem() {
+        final var catalog = new SimpleCatalog();
+
+        final var textWriterBuilder = IonTextWriterBuilder.standard()
+            .withCatalog(catalog)
+            .withCharsetAscii()
+            .withWriteTopLevelValuesOnNewLines(true); // write line separators on new lines instead of spaces
+
+        final var binaryWriterBuilder = IonBinaryWriterBuilder.standard()
+            .withCatalog(catalog)
+            .withInitialSymbolTable(_Private_Utils.systemSymtab(1));
+
+        final var readerBuilder = IonReaderBuilder.standard()
+            .withCatalog(catalog);
+
+        return newLiteSystem(textWriterBuilder, (_Private_IonBinaryWriterBuilder) binaryWriterBuilder, readerBuilder);
+    }
```

Comment:
I suspect this is coming from inside the code of the Ion library?
Can you add a comment explaining from where it comes in case it needs to be updated with upstream changes?